src/components/index/Speaker.astro

```diff
@@ -0,0 +1,82 @@
+---
+
+---
+
+<!-- Will add the feature of going to the next tab in the future -->
+<section class="">
+	<div>
+		<h1 class="text-center text-4xl font-semibold">Keynote Speakers</h1>
```

Comment:
Repeating elements, better to make it into Astro components

Reply:
Done